benches/distributions.rs

```diff
@@ -25,25 +25,12 @@ fn distr_baseline(b: &mut Bencher) {
     b.bytes = size_of::<u64>() as u64 * ::RAND_BENCH_N;
 }
 
-#[bench]
-fn distr_range_int(b: &mut Bencher) {
-    let mut rng = XorShiftRng::new().unwrap();
-    let distr = Range::new(3i64, 134217671i64);
-
-    b.iter(|| {
-        for _ in 0..::RAND_BENCH_N {
-            black_box(distr.sample(&mut rng));
-        }
-    });
-    b.bytes = size_of::<i64>() as u64 * ::RAND_BENCH_N;
-}
-
-macro_rules! distr_range2_int {
+macro_rules! distr_range_int {
     ($fnn:ident, $ty:ty, $low:expr, $high:expr) => {
         #[bench]
         fn $fnn(b: &mut Bencher) {
             let mut rng = XorShiftRng::new().unwrap();
-            let distr = range2::Range::new($low, $high);
+            let distr = range::Range::new($low, $high);
```

Comment:
This can probably be just `Range::new` now. IIRC the only reason for the prefix was because to avoid confusion with the old `range::Range`.

Reply:
Good catch